querybook/webapp/components/QueryEditor/QueryEditor.tsx

```diff
@@ -559,13 +559,11 @@ export const QueryEditor: React.FC<
 
         const handleOnFocus = useCallback(
             (editor: CodeMirror.Editor, event) => {
-                autoCompleter.registerHelper();
```

Comment:
why it's not needed?

Reply:
actually it is needed, otherwise query hint wouldn't work. I changed back the code and added more comments